fiftyone/operators/__init__.py

```diff
@@ -4,6 +4,12 @@
 |
 """
 
+import types
+import wrapt
 from .operator import Operator, OperatorConfig
 from .registry import register_operator
 from .executor import execute_operator
+
+# This enables Sphinx refs to directly use paths imported here
+# and ignore functions marked as depreciated
+__all__ = [k for k, v in globals().items() if not k.startswith("_")]
```

Comment:
Not actually certain this is needed. Let me try again without it.

Reply:
<img width="1469" alt="image" src="https://github.com/voxel51/fiftyone/assets/462228/1cd92699-8603-4270-9eb5-8bfa3be15963">

This is without.

Reply:
With:

<img width="1049" alt="image" src="https://github.com/voxel51/fiftyone/assets/462228/8a5894bc-d876-467a-b8ac-224190f590e1">